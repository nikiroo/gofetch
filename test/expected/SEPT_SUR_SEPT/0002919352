   LA DIRECTRICE GÉNÉRALE D%27ENGIE DÉMENT TOUT PROJET DE CESSION  
                         DU NUCLÉAIRE BELGE                        

  [Economie] 26-09-2018 19:27

  o News link: http://s.7s7.be/3480024
  o Source link: 


    Par: rédaction
    
    26/09/18  - 19h27 Source: Belga    Isabelle Kocher, DG d'Engie.
    © afp.
    
    La  directrice  générale  d'Engie  Isabelle  Kocher  a  démenti
    mercredi  tout  projet  de  vendre  à  EDF  son  parc  de  sept
    réacteurs  nucléaires  en  Belgique, "pays stratégique" pour le
    groupe français selon elle.
    
    "Il  n'y  a  rien  du tout", a-t-elle dit à des journalistes en
    marge  d'un  déplacement à New York, en réponse à un article du
    Canard Enchaîné.
    
    "La  stratégie,  c'est  déjà  de  régler  les sujets techniques
    qu'on  est  en  train  de  traverser"  et "la priorité c'est de
    remettre  ces  centrales  en  ordre  de  marche", a déclaré Mme
    Kocher.
    
    Le  parc  nucléaire  belge,  exploité  par  la  filiale  locale
    d'Engie,    Electrabel,   connaît   en   effet   des   déboires
    actuellement.
    
    La  majorité  des  sept  réacteurs  sont  à  l'arrêt  pour  des
    révisions  dont  la  durée  se prolonge, et selon les autorités
    belges, seul un d'entre eux fonctionnera en novembre.
    
    "La  Belgique,  c'est  un  pays  stratégique  pour Engie. Si on
    arrive  a  accélérer  nos  investissements, puisque la question
    s'était  posée  comme  ça  au  départ,  en  faisant  venir  des
    partenaires, je suis ouverte", a indiqué Isabelle Kocher.
    
    Mais  "ce  n'est  pas d'actualité, il n'y a pas de négociations
    en cours avec qui que ce soit", a-t-elle ajouté.
    
    Selon  le  Canard Enchaîné, Engie aurait aussi proposé à EDF de
    prendre  une  participation  de 49% dans la Compagnie nationale
    du Rhône (CNR), sa filiale d'hydroélectricité.
    
    "Je  ne  sais  pas d'où ça sort... il y a zéro projet de vendre
    la CNR", a assuré la patronne d'Engie.



