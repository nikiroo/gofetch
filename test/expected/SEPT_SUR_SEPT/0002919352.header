0La directrice générale d'Engie dément tout projet de cession du nucléaire belge	null/SEPT_SUR_SEPT/0002919352		70
i  [Economie] 26-09-2018 19:27
i
i    La  directrice  générale  d'Engie  Isabelle  Kocher  a  démenti
i    mercredi  tout  projet  de  vendre  à  EDF  son  parc  de  sept
i    réacteurs nucléaires en ...
i
