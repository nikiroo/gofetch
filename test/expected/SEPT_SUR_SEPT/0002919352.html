<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story'>
	<h2>La directrice générale d%27Engie dément tout projet de cession du nucléaire belge</h2>
	<div class='details'>([Economie] 26-09-2018 19:27)</div>
	<br/>
    <ul>
    <li>Reference: <a href=''>0002919352</a></li>
        <li>News link: <a href='http://s.7s7.be/3480024'>http://s.7s7.be/3480024</a></li>
        <li>Source link: <a href=''></a></li>
    </ul>
	<br/>
	<div class='content' style='text-align: justify'>
		Par: rédaction<br/><br/>26/09/18 - 19h27 Source: Belga    Isabelle Kocher, DG d&#x27;Engie. © afp.<br/><br/>La directrice générale d&#x27;Engie Isabelle Kocher a démenti mercredi tout projet de vendre à EDF son parc de sept réacteurs nucléaires en Belgique, &quot;pays stratégique&quot; pour le groupe français selon elle.<br/><br/>&quot;Il n&#x27;y a rien du tout&quot;, a-t-elle dit à des journalistes en marge d&#x27;un déplacement à New York, en réponse à un article du Canard Enchaîné.<br/><br/>&quot;La stratégie, c&#x27;est déjà de régler les sujets techniques qu&#x27;on est en train de traverser&quot; et &quot;la priorité c&#x27;est de remettre ces centrales en ordre de marche&quot;, a déclaré Mme Kocher.<br/><br/>Le parc nucléaire belge, exploité par la filiale locale d&#x27;Engie, Electrabel, connaît en effet des déboires actuellement.<br/><br/>La majorité des sept réacteurs sont à l&#x27;arrêt pour des révisions dont la durée se prolonge, et selon les autorités belges, seul un d&#x27;entre eux fonctionnera en novembre.<br/><br/>&quot;La Belgique, c&#x27;est un pays stratégique pour Engie. Si on arrive a accélérer nos investissements, puisque la question s&#x27;était posée comme ça au départ, en faisant venir des partenaires, je suis ouverte&quot;, a indiqué Isabelle Kocher.<br/><br/>Mais &quot;ce n&#x27;est pas d&#x27;actualité, il n&#x27;y a pas de négociations en cours avec qui que ce soit&quot;, a-t-elle ajouté.<br/><br/>Selon le Canard Enchaîné, Engie aurait aussi proposé à EDF de prendre une participation de 49% dans la Compagnie nationale du Rhône (CNR), sa filiale d&#x27;hydroélectricité.<br/><br/>&quot;Je ne sais pas d&#x27;où ça sort... il y a zéro projet de vendre la CNR&quot;, a assuré la patronne d&#x27;Engie.
	</div>
<hr/>
</div>
</body>
